src/lib.rs

```diff
@@ -366,4 +375,26 @@ mod contract {
             }
         }
     }
+
+    impl ToStr for crate::types::NonceError {
+        fn to_str(&self) -> &str {
+            match self {
+                Self::NonceOverflow => "ERR_NONCE_OVERFLOW",
+                Self::IncorrectNonce => "ERR_INCORRECT_NONCE",
+            }
+        }
+    }
+
+    trait OrSdkPanic<T, E> {
+        fn unwrap_or_sdk_panic(self) -> T;
+    }
```

Comment:
I think a better name for this, given Rust conventions, would simply be `sdk_unwrap` then at least it will be clear that it behaves like a standard unwrap, but for the SDK.

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/42/commits/d93d0a88a70c2ca33c6d8c4ad02427848af646f3